bsp/worker/src/mill/bsp/worker/MillBuildServer.scala

```diff
@@ -773,3 +775,14 @@ private class MillBuildServer(
     debug("onRunReadStdin is current unsupported")
   }
 }
+
+private object MillBuildServer {
+  private implicit class SeqGroupList[A](private val seq: Seq[A]) extends AnyVal {
+    def groupList[K, B](key: A => K)(f: A => B): Seq[(K, Seq[B])] = {
```

Comment:
Let's move this into a normal static method rather than an extension method. We don't use extension methods much in com-lihaoyi code unless they're super frequently used (e.g. FastParse's `~`), and `groupList` is used in only two places so definitely doesn't make the cut.

GIven that it's only used in two places, and seems pretty niche and specialized to these two callsites, we can probably also hard-code the `(_._2)(_._1)` thing as well

Reply:
Sure